src/vorta/borg/diff.py

```diff
@@ -17,17 +18,17 @@ def prepare(cls, profile, archive_name_1, archive_name_2):
         ret = super().prepare(profile)
         if not ret['ok']:
             return ret
-        else:
-            ret['ok'] = False  # Set back to false, so we can do our own checks here.
-
-        cmd = ['borg', 'diff', '--info', '--log-json']
-        cmd.append(f'{profile.repo.url}::{archive_name_1}')
-        cmd.append(f'{archive_name_2}')
 
+        ret['cmd'] = [
+            'borg', 'diff', '--info', '--log-json',
+            f'{profile.repo.url}::{archive_name_1}',
+            f'{archive_name_2}']
         ret['ok'] = True
-        ret['cmd'] = cmd
+        ret['archive_name_older'] = archive_name_1
+        ret['archive_name_newer'] = archive_name_2
+        ret['json_lines'] = False
+        if borg_compat.check('DIFF_JSON_LINES'):
+            ret['cmd'].insert(4, '--json-lines')
```

Comment:
This is dangerous because the hard-coded position is often overlooked when someone makes changes later. Minor change to get rid of it.